protocol/migrations/sqlite/1632567059_add_highlight_column_in_chats.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE chats ADD COLUMN highlight BOOLEAN DEFAULT TRUE;
```

Comment:
`BOOLEAN DEFAULT TRUE` This will produce unwelcome results, this change will update all existing chats with the highlight field set to `TRUE`. Meaning all chats will be highlighted

Reply:
Great Catch 👏, Thanks